src/shims/unix.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
+fnm exec --using-current -- node "$@"
```

Comment:
May be you should use `exec fnm exec ...` so fnm's process replaces shell and process tree would be like this:

CLI (pid1) -> ~~node shim (pid2)~~ -> fnm (pid2)

Not sure how to do this in windows.

Reply:
oh! TIL!

this is still the problem because we have unknown parents that can bridge it. :(

shell -> scripts/run-tests.sh -> scripts/run-unit-tests.sh -> fnm exec ...

pids don't solve it 😭